inginious/frontend/plugins/show_hints/pages/user_hints.py

```diff
@@ -0,0 +1,124 @@
+
+from inginious.frontend.parsable_text import ParsableText
+
+class UserHint(object):
```

Comment:
`UserHintManager`? and name the file `user_hint_manager`. Also add a better description of what this class is doing.